mountpoint-s3/src/prefetch/checksummed_bytes.rs

```diff
@@ -0,0 +1,250 @@
+use bytes::{Bytes, BytesMut};
```

Comment:
We need to add `pub mod checksummed_bytes;` to `prefetch.rs` for this file to be compiled/tests to run.

Reply:
Ah, my bad, I branched it off from bigger changes and missed that line.